lib/index.js

```diff
@@ -16,8 +57,26 @@ function drawFrog() {
 function draw() {
   ctx.clearRect(0,0, canvas.width, canvas.height);
   drawFrog();
-  x += dx;
```

Comment:
we might need these later for the logs and cars... we will put on the back burner.